### PR TITLE
Add provider seeder

### PR DESCRIPTION
**Pull request by merk**
- Adds a basic provider seeder that adds LC and a fake provider

### CODE REVIEW

**Review by brendonofficial — DISMISSED** [review]
Looks all good 👍

File: tests/Unit/Database/Seeders/ProviderSeederTest.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            new Provider('loyalty-corp', 'Loyalty Corp'),
+            new Provider('moon-corp', 'Mooncorp')
+        ];
+
+        $entityManager = new EntityManagerStub();
+        $seeder = new ProviderSeeder($entityManager);
+
+        $seeder->seed();
+
+        static::assertEquals($expected, $entityManager->getPersisted());
```
Comment (rashmitsingh) [inline tests/Unit/Database/Seeders/ProviderSeederTest.php:33]: If we need to make sure each line is tested, do we also want to make sure we assert flush was called?

File: .phpunit.result.cache [hunk 2]
```diff
@@ -0,0 +1 @@
+C:37:"PHPUnit\Runner\DefaultTestResultCache":665:{a:2:{s:7:"defects";a:0:{}s:5:"times";a:6:{s:93:"Tests\LoyaltyCorp\Multitenancy\Integration\Database\Entities\HasProviderTest::testHasProvider";d:0.047;s:112:"Tests\LoyaltyCorp\Multitenancy\Integration\Database\Entities\HasProviderTest::testHasProviderCanSaveNullProvider";d:0.004;s:90:"Tests\LoyaltyCorp\Multitenancy\Integration\Database\Entities\ProviderTest::testPersistence";d:0.003;s:77:"Tests\LoyaltyCorp\Multitenancy\Unit\Database\Entities\ProviderTest::testState";d:0;s:79:"Tests\LoyaltyCorp\Multitenancy\Unit\Database\Entities\ProviderTest::testNameSet";d:0;s:81:"Tests\LoyaltyCorp\Multitenancy\Unit\Database\Seeders\ProviderSeederTest::testSeed";d:0.004;}}}
```
Comment (rashmitsingh) [inline .phpunit.result.cache]: what is this file for?

**Comment by merk** [comment]
Do not merge: build failure of weird: https://bamboo.loyaltycorp.com.au/browse/PKG-MLTI4-STAN-4/log

File: .phpunit.result.cache (hunk 2 above)
Comment (merk) [reply .phpunit.result.cache]: Weird. Looks like phpunit now outputs a cache?

Comment (rashmitsingh) [reply .phpunit.result.cache]: might be a good idea to add to ignore?

File: tests/Stubs/Vendor/Doctrine/EntityManagerStub.php [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Internal\Hydration\AbstractHydrator;
+use Doctrine\ORM\NativeQuery;
+use Doctrine\ORM\Proxy\ProxyFactory;
+use Doctrine\ORM\Query;
+use Doctrine\ORM\Query\ResultSetMapping;
+use Doctrine\ORM\QueryBuilder;
+use Doctrine\ORM\UnitOfWork;
+
+/**
```
Comment (ketanp77) [inline tests/Stubs/Vendor/Doctrine/EntityManagerStub.php:21]: Missing `@coversNothing`

**Review by ketanp77 — CHANGES_REQUESTED** [review]
Super minor comment

**Pull request merged**